SalesforceMobileSDK-iOS.podspec

```diff
@@ -73,8 +57,8 @@ Pod::Spec.new do |s|
       oauth.dependency 'SalesforceMobileSDK-iOS/SalesforceCommonUtils'
```

Comment:
@khawkins The ```Cocoapods``` merge might interest you (not my area of expertise). You might want to check if I got the new network stuff correct.